service/storage_service.cc

```diff
@@ -610,6 +610,26 @@ void storage_service::mark_existing_views_as_built() {
     }).get();
 }
 
+std::list<gms::inet_address> storage_service::get_ignore_dead_nodes_for_replace() {
+    std::vector<sstring> ignore_nodes_strs;
+    std::list<gms::inet_address> ignore_nodes;
+    boost::split(ignore_nodes_strs, _db.local().get_config().ignore_dead_nodes_for_replace(), boost::is_any_of(","));
```

Comment:
you can strip spaces here too by including a space in `boost::is_any_of(",")` no?
that might create empty tokens but you skip them below anyhow...